Library/Homebrew/bintray.rb

```diff
@@ -30,21 +31,29 @@ def initialize(org: "homebrew")
     ENV["HOMEBREW_FORCE_HOMEBREW_ON_LINUX"] = "1" if @bintray_org == "homebrew" && !OS.mac?
   end
 
-  def open_api(url, *extra_curl_args, auth: true)
-    args = extra_curl_args
-
+  sig { params(url: String, args: String, auth: T::Boolean).returns(SystemCommand::Result) }
+  def open_api(url, *args, auth: true)
     if auth
       raise UsageError, "HOMEBREW_BINTRAY_USER is unset." unless (user = Homebrew::EnvConfig.bintray_user)
       raise UsageError, "HOMEBREW_BINTRAY_KEY is unset." unless (key = Homebrew::EnvConfig.bintray_key)
 
       args += ["--user", "#{user}:#{key}"]
     end
 
-    curl(*args, url,
+    curl(*T.unsafe(args), url,
```

Comment:
Add 

```ruby
include Utils::Curl
```

to the class and change this to

```suggestion
    T.unsafe(self).curl(*args, url,
```